.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: false
+language: node_js
+node_js:
+- '4'
+- '6'
+script:
+- npm run lint
+- npm test
+after_success:
+- npm run coveralls
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: false
+before_script:
+  - npm prune
+after_success:
+  - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
+  - python travis_after_all.py
+  - 'export $(cat .to_export_back) &> /dev/null'
+  - npm run semantic-release
+branches:
+  except:
+    - "/^v\\d+\\.\\d+\\.\\d+$/"
```

Comment:
Yes, just updated that, but would prefer not having to rerun the CLI since it generates a new travis file that I'd have to fiddle with manually (had to last time at least)...